boards/x86/up_squared/doc/index.rst

```diff
@@ -240,49 +181,25 @@ Prepare Linux host
       # tftp
       enable-tftp
       tftp-root=/srv/tftp
-      dhcp-boot=grub_x86_64.efi
+      dhcp-boot=zephyr.efi
 
-   ``grub_x86_64.efi`` is a grub binary created above.
+   ``zephyr.efi`` is a Zephyr EFI binary created above.
 
-#. Create the following directories inside TFTP root :file:`/srv/tftp`
+#. Copy the Zephyr EFI image :file:`zephyr/zephyr.efi` to the
+   :file:`/srv/tftp` folder.
 
     .. code-block:: console
 
-       $ sudo mkdir -p /srv/tftp/EFI/BOOT
-       $ sudo mkdir -p /srv/tftp/kernel
-
-#. Copy the Zephyr image :file:`zephyr/zephyr.strip` to the
-   :file:`/srv/tftp/kernel` folder.
-
-    .. code-block:: console
-
-       $ sudo cp zephyr/zephyr.strip /srv/tftp/kernel
-
-#. Copy your built version of GRUB to :file:`/srv/tftp/grub_x86_64.efi`
-
-#. Create :file:`/srv/tftp/EFI/BOOT/grub.cfg` containing the following:
-
-   .. code-block:: console
-
-      set default=0
-      set timeout=10
+       $ sudo cp zephyr/zephyr.efi /srv/tftp
 
-      menuentry "Zephyr Kernel" {
-         multiboot /kernel/zephyr.strip
-      }
 
 #. TFTP root should be looking like:
 
    .. code-block:: console
 
       $ tree /srv/tftp
       /srv/tftp
-      ├── EFI
-      │   └── BOOT
-      │       └── grub.cfg
-      ├── grub_x86_64.efi
-      └── kernel
-          └── zephyr.strip
+      ├── zephyr.efi
```

Comment:
If we want to have a directory tree we need to use "└──"

Reply:
thanks, I have updated it.